src/OpenTracing/Tag.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace OpenTracing;
+
+class Tag
+{
+
+    const PEER_SERVICE = "peer.service";
+    const PEER_HOSTNAME = "peer.hostname";
+    const PEER_HOST_IPV4 = "peer.ipv4";
+    const PEER_HOST_IPV6 = "peer.ipv6";
+    const PEER_PORT = "peer.port";
```

Comment:
I don't recall how good PHP is with classes and type validation, if it can catch passing the wrong value type into a method (e.g. string to method expecting integer), then instead of string tags it's better to use the pattern from Go.